### ISSUE TITLE
Color accessibility review

### ISSUE THREAD

**Issue by opattison** [opattison/olivermakes#304]
Addressing in #302

Shades should be scaled using SCSS functions. In some cases, shades need to be a bit darker or lighter. Links should be higher contrast (darker than text in the `base` theme) with accent underlines (in `$red`).

<img width="749" alt="2016-03-25 11 26 12" src="https://cloud.githubusercontent.com/assets/885300/14047158/6ccec986-f27c-11e5-92da-16e4c52aab87.png">
